src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Automation/Peers/AutomationPeer.cs

```diff
@@ -1263,6 +1277,58 @@ public int GetSizeOfSet()
             return result;
         }
 
+        /// <summary>
```

Comment:
I think you should fill the xml doc, similar to what you did in IsDialog.

Reply:
Done